components/formats-gpl/src/loci/formats/in/SlideBook6Reader.java

```diff
@@ -81,15 +82,14 @@
 	private static boolean libraryFound = false;
 
 	static {
-		String library_path = System.getProperty("java.library.path");
-
 		try {
 			// load JNI wrapper of SBReadFile.dll
-			System.loadLibrary("SlideBook6Reader");
-			libraryFound = true;
+			if (!libraryFound) {
+				libraryFound = NativeLibraryUtil.loadNativeLibrary(SlideBook6Reader.class, "SlideBook6Reader");
+			}
 		}
 		catch (UnsatisfiedLinkError e) {
-			LOGGER.debug(NO_3I_MSG, e);
+			LOGGER.warn(NO_3I_MSG, e);
```

Comment:
This should remain at ```debug```, otherwise a warning will be printed every time a file is initialized without the .dll present - even for non-Slidebook files.